### ISSUE TITLE
phantomsetup: bug with reading the example MESA file

### ISSUE THREAD

**Issue by danieljprice** [danieljprice/phantom#291]
using SETUP=star the default setup for a MESA star uses the sample datafile:

data/star_data_files/P12_Phantom_Profile.data

...but currently gives an error when reading this file. This should also be tested as part of the actions suite, and/or better default data files supplied.

**Comment by themikelau** [comment]
Happy to take a look at that.

**Comment by themikelau** [comment]
Right now, the following IF statement is triggered by the column name `#[Mass]`, which is read as `#mass`:
```
if (header(i)(1:1) == '#') then
       print '("Detected wrong header entry : ",A," in file ",A)',trim(lcase(header(i))),trim(fullfilepath)
       ierr = 2
       return
    endif
```
This IF statement assumes that header lines cannot begin with "#", conflicting with the format of P12_Phantom_Profile.data. I personally have never encountered the need for this IF statement before, since read_mesa() always finds the correct header line.

**Comment by danieljprice** [comment]
fixed in #297
